### PR TITLE
Removed sqlservernotebook references as it no longer exists

### PR DESCRIPTION
**Pull request by smartguest**
It is not found in the extensions folder, so it should be removed.

### CODE REVIEW

File: product.json [hunk 1]
```diff
@@ -78,12 +78,6 @@
 	"extensionsGallery": {
 		"serviceUrl": "https://sqlopsextensions.blob.core.windows.net/marketplace/v1/extensionsGallery.json"
 	},
-	"builtInExtensions": [
-		{
-			"name": "Microsoft.sqlservernotebook",
```
Comment (kburtram) [inline product.json:83]: @chlafreniere do you have context on if it's fine to remove this extension reference?

**Review by Charles-Gagnon — CHANGES_REQUESTED** [review]
builtInExtensions are downloaded from the marketplace during build time - this should not be removed.

The localization files are slightly different though. Given that this extension is built into the product it makes some amount of sense to include them in the ADS language packs instead of bundling with the extension itself. Of course this then makes it more difficult to actually sync up with any changes made to the actual things being localized in the extension. So for that it probably makes sense to just bundle the localization files with the extension itself instead.

Do we even have a process for the localization team to go through extensions that aren't in the main repo?

**Comment by smartguest** [comment]
@Charles-Gagnon Ah I see, I'll reinclude the Microsoft.sqlservernotebook extension back into my lang pack, didn't know it was a generated type extension.

**Comment by Charles-Gagnon** [comment]
I don't know how useful having the localized files in this repo is though - we should look into moving them into the actual source repo since right now it's mostly just going to be wrong. 

And anyways I don't even see the localized JSON's being included in the lang packs - so I think something is missing there anyways which means that even with these files here the extension isn't actually localized when installing the lang packs.

**Pull request closed without merging**